spec/companies_house/client_spec.rb

```diff
@@ -307,6 +307,91 @@
     end
   end
 
+  describe "#persons_with_significant_control_statements" do
+    include_context "test client"
+
+    subject(:response) do
+      client.persons_with_significant_control_statements(company_id, register_view: register_view)
+    end
```

Comment:
This should fix the rubocop error

```suggestion
    subject(:response) do
      client.persons_with_significant_control_statements(company_id, register_view: register_view)
    end
    
    include_context "test client"
```